releasenotes/notes/add-blazar-fa6ecce8b21c73b4.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I've pulled this one from yoga as I've retrospectively enabled blazar in an earlier release. Hope this doesn't cause reno to barf.